crates/miden-block-prover/src/local_block_prover.rs

```diff
@@ -15,43 +18,29 @@ impl LocalBlockProver {
         Self {}
     }
 
-    /// Proves the provided [`SignedBlock`] into a [`ProvenBlock`].
-    ///
-    /// For now this does not actually verify the batches or create a block proof, but will be added
-    /// in the future.
+    /// Executes a proof of a block in the chain based on the given header and inputs.
```

Comment:
> Executes a proof

Nit: Proofs cannot be executed, so was the intention to write something different here?